Tasks/ANT/anttask.ts

```diff
@@ -208,39 +219,28 @@ async function doWork() {
 
         var ccTool = tl.getInput('codeCoverageTool');
         var isCodeCoverageOpted = (typeof ccTool != "undefined" && ccTool && ccTool.toLowerCase() != 'none');
-
         var buildRootPath = path.dirname(antBuildFile);
         var instrumentedClassesDirectory = path.join(buildRootPath, "InstrumentedClasses");
-        //delete any previous cobertura instrumented classes as they might interfere with ant execution.
-        tl.rmRF(instrumentedClassesDirectory, true);
-
-        if (isCodeCoverageOpted) {
-            var summaryFile: string = null;
-            var reportDirectory: string = null;
-            var ccReportTask: string = null;
-            var reportBuildFile: string = null;
-            enableCodeCoverage();
-        }
-        else {
-            tl.debug("Option to enable code coverage was not selected and is being skipped.");
-        }
-
+        var summaryFile: string = null;
+        var reportDirectory: string = null;
+        var ccReportTask: string = null;
+        var reportBuildFile: string = null;
         var publishJUnitResults = tl.getInput('publishJUnitResults');
         var testResultsFiles = tl.getInput('testResultsFiles', true);
 
-        await antv.exec();
+        //delete any previous cobertura instrumented classes as they might interfere with ant execution.
+        tl.rmRF(instrumentedClassesDirectory, true);
```

Comment:
dont rely on `true` to continue on error. it doesn't appear to be wired up. try/catch instead.